src/quantuminspire/projectq/backend_qx.py

```diff
@@ -331,7 +331,8 @@ def _run(self):
             if self._perform_execution:
                 self._quantum_inspire_result = self.quantum_inspire_api.execute_qasm(
                     self._cqasm, number_of_shots=self._num_runs, backend_type=self.backend_type)
-                if len(self._quantum_inspire_result.get('histogram', {})) == 0:
+
+                if not self._quantum_inspire_result or not self._quantum_inspire_result.get('histogram', {}):
```

Comment:
Why `not self._quantum_inspire_result.get('histogram', {})` instead of `len(self._quantum_inspire_result.get('histogram', {})) == 0` ? We still want to bark at an empty histogram, right? That has not changed in the api, and if there is no histogram we will still get {} I think?

Reply:
✔️